packages/core/src/__integrationtests__/AccountLinking.spec.ts

```diff
@@ -172,11 +224,14 @@ describe('When there is an on-chain DID', () => {
           Did.Utils.signatureAlgForKeyType[keyType]
         )
         keypair = keyTool.keypair
+        keypairChain = AccountLinks.accountToChain(keypair.address)
         sign = AccountLinks.makeLinkingSignCallback(keypair)
         didKey = makeSigningKeyTool()
         newDidKey = makeSigningKeyTool()
         did = await createFullDidFromSeed(paymentAccount, didKey.keypair)
+        didChain = Did.Chain.didToChain(did.uri)
```

Comment:
the namespacing gets a little funny here, but afaik namespacing is an issue we want to tackle separately?

Reply:
Exactly, and my proposal already addresses this particular case